doc/ubuntu_release_process.md

```diff
@@ -37,7 +37,7 @@ new-upstream-release does:
   * makes changes to debian/patches/series and drops any cherry-picks in that directory.
   * refreshes any patches in debian/patches/
 
-  * opens $EDITOR with an option to edit debian/changelog.  In SRU, I will generally strip out commits that are not related to ubuntu, and also strip out or join any fix/revert/fixup commits into one.  Note this is a *ubuntu* changelog, so it makes sense that it only have Ubuntu specific things listed. 
+  * opens $EDITOR with an option to edit debian/changelog.  In SRU, I will generally strip out commits that are not related to ubuntu, and also strip out or join any fix/revert/fixup commits into one.  Note this is a *ubuntu* changelog, so it makes sense that it only have Ubuntu specific things listed.
```

Comment:
Here's the note ryan that maybe we want to edit? This was the reason I had manually redacting debian/changelog lines that didn't make sense for the debian package (like dropping any tests/github or FreeBSD-related commit entries)

Do we want to change this statement to say that any commits that need redacting should be filtered automatically by log2dch?

Reply:
Yes, we can do that separately;  this is just a whitespace change.